localboot/main.go

```diff
@@ -123,7 +124,20 @@ func BootGrubMode(devices []storage.BlockDev, baseMountpoint string, guid string
 	if len(bootconfigs) == 0 {
 		return fmt.Errorf("No boot configuration found")
 	}
-
+	for n, cfg := range bootconfigs {
+		log.Printf("  %d: %s\n", n, cfg.Name)
+	}
+	if configNo > -1 {
```

Comment:
this changes the current behaviour - from booting the first entry, to trying all the boot entries until one works. This is also different from Grub's behaviour. I would just remove this, and change the default index to 0 as suggested above

Reply:
I don't understand. The purpose of this flag schould be to choose one specific bootentry from the ones found - which indeet changes the usual behavior. If you don't use the flag the current behavior is applied: trying alle bootconfigs until oneworks

Reply:
my bad, scratch that request. We had in store to only try to boot the first entry but we never implemented it, so you're not changing the default behaviour as I initially thought.

(my other requests still apply though)

Reply:
ok ;)
but I will leave the -1 as default, to detect if the flag is used at all.